rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -82,6 +82,7 @@
     private static final String PROPERTY_RPC_WEBSOCKET_ENABLED = "rpc.providers.web.ws.enabled";
     private static final String PROPERTY_RPC_WEBSOCKET_ADDRESS = "rpc.providers.web.ws.bind_address";
     private static final String PROPERTY_RPC_WEBSOCKET_PORT = "rpc.providers.web.ws.port";
+    private static final String PROPERTY_RPC_WEBSOCKET_SERVER_WRITE_TIMEOUT = "rpc.providers.web.ws.server_write_timeout";
```

Comment:
If the variable name for `serverWriteTimeout` is to be updated to include units, don't forget to update it here too.

Reply:
renamed